config/locales/de.yml

```diff
@@ -162,157 +227,141 @@ de:
       update_needs_confirmation: Ihre Daten wurden aktualisiert, aber Sie müssen Ihre neue Mailadresse bestätigen. Sie erhalten in wenigen Minuten eine Email, mit der Sie die Änderung Ihrer Mailadresse abschliessen können.
       updated: Ihre Daten wurden aktualisiert.
     sessions:
+      already_signed_out: Erfolgreich abgemeldet.
       signed_in: Erfolgreich angemeldet.
       signed_out: Erfolgreich abgemeldet.
-      already_signed_out: Erfolgreich abgemeldet.
     unlocks:
       send_instructions: Sie erhalten in wenigen Minuten eine Email mit der Anleitung, wie Sie Ihren Account entsperren können.
       send_paranoid_instructions: Falls Ihre Mailadresse in unserer Datenbank existiert erhalten Sie in wenigen Minuten eine Email mit der Anleitung, wie Sie Ihren Account entsperren können.
       unlocked: Ihr Account wurde entsperrt. Sie sind jetzt angemeldet.
+  edit_login: Login bearbeiten
+  edit_profile: Profil bearbeiten
   errors:
     messages:
-      blank: darf nicht leer sein
       already_confirmed: wurde bereits bestätigt
+      blank: darf nicht leer sein
       confirmation_period_expired: 'muss innerhalb %{period} bestätigt werden, bitte fordern Sie einen neuen Link an'
       expired: ist abgelaufen, bitte neu anfordern
       not_found: nicht gefunden
       not_locked: ist nicht gesperrt
       not_saved:
         one: 'Konnte %{resource} nicht speichern: ein Fehler.'
         other: 'Konnte %{resource} nicht speichern: %{count} Fehler.'
+  forgot_password: Passwort vergessen?
+  gender:
+    female: Weiblich
+    male: Männlich
   helpers:
     submit:
       create: '%{model} erfassen'
+      destroy: '%{model} löschen'
+      read: '%{model} anzeigen'
+      show: '%{model} anzeigen'
       update: '%{model} aktualisieren'
       user:
-        update: Login aktualisieren
+        create: Benutzer erstellen
+        update: Update login
+  invite_sent: 'Einladung wurde an %{email} verschickt.'
+  language_level:
+    basic: Grundlagen
+    fluent: Fliessend
+    good: Gut
+    native_speaker: Muttersprachler
+  language_skills: Sprachkenntnisse
+  login: Anmelden
+  logout: Abmelden
+  not_authorized: Sie sind nicht berechtigt diese Aktion durchzuführen.
+  password_reset_instructions: Ich möchte mein Passwort zurücksetzen
+  personal_background: Persönlicher Hintergrund
+  profile_created: Profil wurde erfolgreich erstellt.
+  profile_updated: Profil wurde erfolgreich aktualisiert.
+  registered_by: 'SozialberaterIn'
```

Comment:
ticks not needed. In general, extensive changes in the translation strategy or order could be in a separate branch.